gradle.properties

```diff
@@ -16,3 +16,4 @@ POM_DEVELOPER_ID=jakewharton
 POM_DEVELOPER_NAME=Jake Wharton
 
 org.gradle.jvmargs=-Xmx1536M
+android.enableD8.desugaring=true
```

Comment:
I don't think we need this. We don't ship an APK anywhere.

Reply:
Ok with removing it, but I thought this would apply D8's version of J8->J7 bytecode transforming?

Reply:
Only when building an APK. Aars don't ship dexes so there's no opportunity for it to run. Plus the lint rules always run on JDK 8 or newer in AS or Gradle.

Reply:
oh duh, good point.

Reply:
removed